.pylintrc

```diff
@@ -23,4 +24,5 @@ indent-string='    '
 max-line-length=160
```

Comment:
Can we add an exception to just the one line if it's not possible to break it down? 160 is very long.

Reply:
This exception is from a different PR(PR No:663).
https://github.com/aws-cloudformation/cloudformation-cli/pull/663/files#diff-ba293004ae8f18975051e825bd8d5c70137a8b5d42d16e0f7c9354aa634c446c